beekeeper-core/pom.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure if we should move all the common path-cleaner stuff to the core module. The core module is used by scheduler and everything else which means the clean up stuff will be available to pretty much every other beekeeper module we make. 

I think it might be a good idea to follow the same pattern we have for the scheduler. So `beekeeper-scheduler` is the generic module for scheduling an entity and then `beekeeper-scheduler-apiary` uses that module to schedule for apiary events. Similarly, we could have the `beekeeper-cleanup` module with the common clean up stuff and then `beekeeper-cleanup-path` and `beekeeper-cleanup-metadata` make use of the clean up module. What do you think?

@massdosage @patduin @abhimanyugupta07

Reply:
Yeah thats a good point, and would make sense to do it in the same way as the scheduler stuff.

Reply:
Sounds good !

Reply:
+1